packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -199,3 +204,9 @@
 .edit-site-global-styles-sidebar__panel .block-editor-block-icon svg {
 	fill: currentColor;
 }
+
+.edit-site-global-styles-type-preview {
```

Comment:
are these styles something that can be absorbed into the components? This file starts to feel a little bit hectic

Reply:
Might be a good janitorial follow up to move variation components into `components/global-styles/variations` and then all the related CSS as well?